tests/run/builtin_abs_overflow.srctree

```diff
@@ -0,0 +1,50 @@
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import a; assert a.test() == 3"
```

Comment:
This doesn't need a `.srctree` test. Just drop a plain `builtin_abs_overflow.pyx` file into `tests/run/`, and add a doctest as docstring to the `test()` function that calls it. Or, in fact, make that three different functions, so that a test failure makes it clearer what failed.